src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,108 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int PRIME1 = 1;
+    private static final int PRIME2 = 17;
+    private int capacity;
```

Comment:
There is no need for this variable. If we need to recalculate the new length, it is enough to use length

Reply:
Not fixed